packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/add-orgs.ts

```diff
@@ -0,0 +1,106 @@
+import test, { Test } from "tape-promise/tape";
```

Comment:
@RafaelAPB Apologies I just spotted this, please add a `.test` suffix to the filename e.g. `add-org.test.ts`